### DISCUSSION TITLE
Show a User Computed Value in User Slot Graph

### DISCUSSION THREAD

**Discussion by ThinAirPhoto** [aaronwmorris/indi-allsky#1556] [Q&A]
Up and running with a Raspberry Pi HQ setup. Cannot say enough about how awesome this software is...

Is there a way to display a computed value in one of the User Slot Graphs? Specifically, I wanted to try showing sky brightness in magnitudes/square arcsec instead of (or in addition to) the jSQM value that is already shown. I think I have found the equations to do the conversion, but no idea if this is possible, or if so how to go about mechanizing it...

Thanks bunches,

ML

**Comment by aaronwmorris** [comment]
I have been sitting on the SQM stuff for a while.  The jSQM value was supposed to be in magnitudes, but I just left the ADU value as a placeholder.  Let me see if I can carve out some time to focus on that.

If you have suggestions for the equation I would be happy to hear it.

**Reply by ThinAirPhoto** [reply]
Aaron,

Well, when I first posed this question I had found some equations that looked to be converting between the 2 units. But now going back to the web, I cannot find that reference anymore. I did find a spreadsheet that goes through a calculation starting with a background ADU value and ending up giving out mpsa, and by reference, then NELM and Bortle too. It is MUCH more complex than what I initially found, although I am now suspecting it is likely more accurate, as it seems to take into account a number of relevant factors that the original equations I saw did not worry about. The link to it is here if you want to check it out:

[https://stargazerslounge.com/topic/310533-sky-background-brightness-calculator/](url)

I have a good friend whose area of expertise is in this area. I will see what he says about it. It may be that by the time you have to take all the factors into consideration required to make an accurate calculation it is too much trouble...In any case, thanks for getting back to me and being willing to look into it. Top notch support for a top notch app for sure.
